apps/cms/lib/repo.ex

```diff
@@ -29,6 +33,8 @@ defmodule CMS.Repo do
 
   @cms_api Application.get_env(:cms, :cms_api)
 
+  @ttl :timer.hours(1)
```

Comment:
We set a TTL because we don't have a sophisticated mechanism to handle failed requests from Drupal to Dotcom.

Reply:
question: If you're increasing the TTL to an hour, do need any of the Nebulex changes? It seems like that alone would make a big difference in the number of cache hits.

Reply:
it will, but the content creators need the ability to reset a key so that content gets pushed more quickly. the ttl has been one minute so that they see the content change quickly. this lets us actually cache the content, but also update it quickly...something we can't do without significant work using ets and elixir nodes that don't talk to each other currently.

Reply:
the ideal state would actually to use a message queue instead of sending patch requests. now that i know terraform...lookout. ultimately, there is no way around the two generals problem. so setting some ttl is a safety net.